src/math/SpatialCell.cc

```diff
@@ -86,9 +86,8 @@ SpatialCell::SpatialCell(std::string const& label, ///< string representing "nam
     _candidateList(candidateList),
     _ignoreBad(true)
 {
-    lsst::pex::logging::TTrace<3>("lsst.afw.math.SpatialCell",
-                                  "Cell %s : created with %d candidates",
-                                  this->_label.c_str(), this->_candidateList.size());
+    LOGL_DEBUG("TRACE2.afw.math.SpatialCell", "Cell %s : created with %d candidates",
+               this->_label.c_str(), this->_candidateList.size());
```

Comment:
I also don't think the `TRACE2` prefix is needed here.

I may find an exception, but in general I think `afw.math` is enough of a smorgasbord of different things that no one will ever want to enable logging on all `afw.math.*` at a particular level; they'll only care about enabling individual named sub-logs.